.github/workflows/release-baseci-testcontainers.yml

```
@@ -0,0 +1,36 @@
+name: Release BaseCI Testcontainers
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "baseci-testcontainers/*"
+  workflow_dispatch:
+
+
+jobs:
+  release-baseci:
```

Update the job name just to avoid conflict
```suggestion
  release-baseci-testcontainers:
```